docs/glossary/terms.md

```diff
@@ -0,0 +1,226 @@
+---
+sidebar_position: 1
+sidebar_label: Terms
+---
+
+## anyone-can-pay (ACP)
```

Comment:
Why are the initial letters of the terms all in lowercase? Should they be capitalized instead?
For example: "anyone-can-pay (ACP)" -> "Anyone-can-pay (ACP)"

Reply:
Regarding capitalization there's no strict rule. The key is to keep the style consistent.

Reply:
I vote for capitalizing the first letter of every term, the current mix is awkward